airbyte-integrations/connectors/destination-redshift/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.25.0'
     features = ['db-destinations', 's3-destinations', 'typing-deduping']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
We don't want to commit this change, correct?

Reply:
Nope